tests/test_local_move.py

```diff
@@ -222,8 +230,12 @@ def test_local_md_particle_density():
     for p, bp in zip(sys_params, unbound_potentials):
         bps.append(bp.bind(p).bound_impl(np.float32))
 
-    def observable(x):
-        ixns = nblist.get_nblist(coords, box, cutoff)
+    def observable(pair):
```

Comment:
nit: a more descriptive name and/or comment seems useful here (took me a minute to understand what the observable was)

Reply:
Changed in [04d13a6](https://github.com/proteneer/timemachine/pull/968/commits/04d13a6dd6e434582e7c327287c282bd3d9fe7fb)